pr_testing/test_multiple_prs.sh

```diff
@@ -1196,9 +1196,18 @@ for BT in ${ENABLE_BOT_TESTS}; do
              echo "<li><a href=\"$PROFILING_WORKFLOW/\">$PROFILING_WORKFLOW/</a> </li>" >> $WORKSPACE/upload/profiling/index.html
              get_jenkins_artifacts igprof/${CMSSW_IB}/${ARCHITECTURE}/profiling/${PROFILING_WORKFLOW}/RES_CPU_step3.txt  ${CMSSW_IB}_RES_CPU_step3.txt || true
              $WORKSPACE/profiling/Analyze_tool/compare_cpu_txt.py --old ${CMSSW_IB}_RES_CPU_step3.txt --new RES_CPU_step3.txt > RES_CPU_compare_$PROFILING_WORKFLOW.txt || true
-             mkdir -p $WORKSPACE/upload/profiling/
-             cp -p RES_CPU_compare_$PROFILING_WORKFLOW.txt $WORKSPACE/upload/profiling/$d/ || true
              echo "<li><a href=\"$PROFILING_WORKFLOW/RES_CPU_compare_$PROFILING_WORKFLOW.txt\">Igprof Comparison cpu usage RECO produce methods.</a> </li>" >> $WORKSPACE/upload/profiling/index.html
+             cp $WORKSPACE/cms-bot/comparisons/compareProducts.* ./
+             if (get_jenkins_artifacts igprof/${CMSSW_IB}/${ARCHITECTURE}/profiling/${PROFILING_WORKFLOW}/step3_sizes_${PROFILING_WORKFLOW}.txt  ${CMSSW_IB}_step3_sizes_${PROFILING_WORKFLOW}.txt) ; then
+               edmEventSize -v step3*.root > step3_sizes_${PROFILING_WORKFLOW}.txt
```

Comment:
is there any chance that this command might fail? If yes then add `|| true` at the end so that it does not break the whole comparison test

Reply:
Done.